examples/persistable.php

```diff
@@ -106,4 +106,4 @@ public function bsonUnserialize(array $data): void
 $foundEntry = $collection->findOne([]);
 
 /** @psalm-suppress ForbiddenCode */
```

Comment:
ForbiddenCode suppressions can be removed as only `var_dump` and `shell_exec` are included by default: https://psalm.dev/docs/running_psalm/configuration/#forbiddenfunctions

I tested this locally and confirmed.

Reply:
Done. This further impacts my lines added < lines deleted ratio. 😂